apps/cli/src/config.ts

```diff
@@ -0,0 +1,409 @@
+import bytes from "bytes";
+import os from "os";
+import { extname } from "path";
+import { TomlPrimitive, parse as parseToml } from "smol-toml";
+
+/**
+ * Configuration for drives of a Cartesi Machine. A drive may already exist or be built by a builder
+ */
+const DEFAULT_FORMAT = "ext2";
+const DEFAULT_RAM = "128Mi";
+const DEFAULT_RAM_IMAGE_DOCKER = "/usr/share/cartesi-machine/images/linux.bin";
+const DEFAULT_RAM_IMAGE_LINUX = "/usr/share/cartesi-machine/images/linux.bin";
+const DEFAULT_RAM_IMAGE_MAC =
+    "/opt/homebrew/share/cartesi-machine/images/linux.bin";
+const DEFAULT_SDK = "cartesi/sdk:0.10.0";
+
+type Builder = "directory" | "docker" | "empty" | "none" | "tar";
+type DriveFormat = "ext2" | "sqfs";
+
+export type DirectoryDriveConfig = {
+    builder: "directory";
+    extraSize: number; // default is 0 (no extra size)
+    format: DriveFormat;
+    directory: string; // required
+};
+
+export type DockerDriveConfig = {
+    builder: "docker";
+    dockerfile: string;
```

Comment:
I miss a `buildContext` parameter.

Imagine I want to build two drives using `buidler: docker` but using differente subdiretories in my project.

I wan't to use this to achieve the `cruntime` that I made in the past just using the new `cartesi.toml` format.